Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.11.6-alpine3.18
+
+ENV PYTHONDONTWRITEBYTECODE=1
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+COPY requirements.txt .
```

Comment:
Consider using a specific path for the `COPY requirements.txt .` command to avoid potential conflicts with other files named `requirements.txt` in different directories.